### PR TITLE
Player 552 b

### PR DESCRIPTION
**Pull request by pilievOoyala**

### CODE REVIEW

File: js/google_ima.js [hunk 1]
```diff
@@ -279,6 +279,12 @@ require("../html5-common/js/utils/utils.js");
           this.useInsecureVpaidMode = metadata.vpaidMode === "insecure";
         }
 
+        this.disableFlashAds = false;
```
Comment (aeng7) [inline js/google_ima.js]: Should we also define this and set to false in the _resetVars function? The other page level settings are treated like this.

File: test/unit-test-helpers/mock_ima.js [hunk 2]
```diff
@@ -64,7 +64,8 @@ google =
       setPlayerVersion : function() {},
       setPlayerType : function() {},
       setVpaidMode : function() {},
-      setLocale : function() {}
+      setLocale : function() {},
+      setDisableFlashAds : function() {},
```
Comment (aeng7) [inline test/unit-test-helpers/mock_ima.js]: Can we add a unit test to check that the page level setting reaches this mock setDisableFlashAds function with the correct value?

File: js/google_ima.js (hunk 1 above)
Comment (pilievOoyala) [inline js/google_ima.js]: I totally forgot about that. good catch.

File: test/unit-tests/ima_test.js [hunk 3]
```diff
@@ -1512,4 +1512,28 @@ describe('ad_manager_ima', function()
     videoWrapper.destroy();
     expect(ima.sharedVideoElement).to.be(null);
   });
+
+  it('Test disabling flash ads flag', function()
+  {
+    var flag;
+    google.ima.settings.setDisableFlashAds = function(value) {flag = value};
```
Comment (pilievOoyala) [inline test/unit-tests/ima_test.js]: I need to remove this. I rushed this test out.

File: test/unit-test-helpers/mock_ima.js (hunk 2 above)
Comment (pilievOoyala) [inline test/unit-test-helpers/mock_ima.js]: Added below.  I dont think I can run the actual ad request code, where we set the param in the ima sdk but I can test that loadmetadata is reading it in correctly.

**Comment by aeng7** [comment]
LGTM

**Pull request merged**